File: Day 8-Dictionaries and Maps/Solution.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
         Scanner scan = new Scanner(System.in);
         int n = scan.nextInt();
         Map<String, Integer> phoneBook = new HashMap<String, Integer>();
            for(int i = 0; i < n; i++){
                String name = scan.next();
                int phone = scan.nextInt();
                phoneBook.put(name, phone);
            }
            while(scan.hasNext()){
                String name = scan.next();
                if(phoneBook.containsKey(name)){
                    System.out.println(name + "=" + phoneBook.get(name));
                } else {
                    System.out.println("Not found");
                }
            }
    }
}

File: Day 8-Dictionaries and Maps/Solution.kt
fun main(args: Array<String>) {
    val n = readLine()!!.toInt()
    val phoneBook = mutableMapOf<String, String>()
    for (i in 0 until n) {
        val (name, phone) = readLine()!!.split(" ")
        phoneBook[name] = phone
    }
    var name: String?
    while (readLine().also { name = it } != null) {
        if (phoneBook.containsKey(name)) {
            println("$name=${phoneBook[name]}")
        } else {
            println("Not found")
        }
    }
}